kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/BaseSite.java

```diff
@@ -837,7 +837,7 @@ public User getModifiedBy()
 	public Date getCreatedDate() {
 		Date date = null;
 		if (m_createdTime != null) {
-			new Date(m_createdTime.toEpochMilli());
+			date = new Date(m_createdTime.toEpochMilli());
```

Comment:
return m_createdTime != null ? new Date(m_createdTime.toEpochMilli()) : null;

Reply:
Have pushed this change and also applied it to other method